dotCMS/src/main/java/com/dotcms/cube/filters/SimpleFilter.java

```diff
@@ -64,7 +64,7 @@ public Operator getOperator() {
     }
 
     public String[] getValues() {
-        return values;
+        return Arrays.stream(values).map(Object::toString).toArray(String[]::new);
```

Comment:
be defensive, check if values is not before transformation

Reply:
done https://github.com/dotCMS/core/pull/24625/commits/989229988f13072ff026d0e585bd2ddb61304cc3#diff-b9cd675fa490c8ec6e6ea7c221ee177d2481c6d60070f747e70316e045201a2fR67